### ISSUE TITLE
Playtesting - Rent system is poorly and wrongly explained in the available hint.

### ISSUE THREAD

**Issue by Nextej** [loiste-interactive/Obenseuer-Issues#990]
- The hint states that the rent is apartment value with added basic income from the tenant, which is untrue because it's the 1/100 of the apartment value in RMs for each currency counted separately. So it's for OC income it's the Apartment Value in RMs * 1/100 * RM-to-OC conversion + basic OC income; for RM it's the Apartment Value in RMs * 1/100 + basic RM income. Either adjust the displayed value of the apartments or state that it's the 1/100 of the apartment value.
- The second issue with the description is that it doesn't account for the fact that if a tenant doesn't receive RM income, the value RM is not counted at all (or other way around, but I don't think there's anyone who pays only in RMs), so it should state that "If the resident does not receive income in specific currency, the rent in that currency is not paid at all".
- Currently the explanatory hint is available only at renting UI screen, while it should be displayed all the time in the tenement tab, should players have any doubts about how it is counted after they've tenned the residents.
- Would be nice to also be able to see what the basic income of the residents is after renting them the apartment, since if you haven't noted that down beforehand, you got no way of knowing what the basic income is unless you know how it is calculated (which again the descriptions fails at).
- Rented apartments with residents that don't pay RMs could have a apartment value displayed in OCs instead and/or add ability to display apartment value in OC additionally to every apartment (perhaps by hovering over RM value would have a pop-up saying "Value: About <value> OC")

**Comment by Orphiex** [comment]
Agreed; I've been writing up a tenement guide listing the renters, their wishlists, and the rent earnable, but I've been holding off posting it because I didn't understand the relative income formulas. This explanation clears up a lot, but it should definitely be clearer in-game.
